### ISSUE TITLE
Support git push options 

### ISSUE THREAD

**Issue by ambroisie** [alarsyo/lohr#3]
Motivating example: [Sourcehut's push options](https://man.sr.ht/git.sr.ht/#push-options) includes a way to set the repository's visibility when pushing to it. Lohr could either try to be smart and set the corresponding option automatically when pushing a repository, or it should be configurable using the `.lohr` file.

I'm more partial to it being set as a general config, as I would rather avoid having to write `.lohr` files.

**Comment by alarsyo** [comment]
How would this work for a general config option, though? Setting all repos to the same visibility doesn't sound like what we want

**Comment by ambroisie** [comment]
I was thinking that if a repo is marked as private, it would push it as private.

**Comment by alarsyo** [comment]
So something like an explicit list of private repos ? 

```yaml
private_repos:
- "full/name"
```

Edit: probably something more generic would be more appropriate, so:

```yaml
push_options:
- option: "visibility=private"
  remotes:
  - "git@git.remote.net:full/name"
```

**Comment by ambroisie** [comment]
No, you can just look at the visibility of a repository by looking at the payload from the webhook.
